test/cloudtest/pkg/providers/packet/packet_cleanup/tool.go

```diff
@@ -0,0 +1,188 @@
+// Copyright (c) 2019 Cisco Systems, Inc and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package main - Packet cleanup utils
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/networkservicemesh/networkservicemesh/test/cloudtest/pkg/providers/packet/packethelper"
+
+	"github.com/packethost/packngo"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+type packetCleanupCmd struct {
+	cobra.Command
+
+	cmdArguments *Arguments
+}
+
+// Arguments - command line arguments
+type Arguments struct {
+	clusterPrefix   []string
+	sshPrefix       string
+	token           string
+	projectID       string
+	clusterLifetime time.Duration
+	deleteSSHKeys   bool
+	deleteClusters  bool
+}
+
+func initCmd(rootCmd *packetCleanupCmd) {
+	rootCmd.Flags().BoolVarP(&rootCmd.cmdArguments.deleteSSHKeys, "ssh", "k", false, "Delete ssh keys")
+	rootCmd.Flags().BoolVarP(&rootCmd.cmdArguments.deleteClusters, "clusters", "c", false, "Delete ssh keys")
+	rootCmd.Flags().DurationVarP(&rootCmd.cmdArguments.clusterLifetime, "since", "s", 4*time.Hour, "Since timeout, if exceed ssh/cluster will be deleted")
```

Comment:
'since' usually applied to some point in time, but the argument meaning is duration
IMO, more descriptive name needed, something like "older-than", "age-limit", "age-threshold", "max-age"